src/action-creators/index.js

```diff
@@ -18,6 +18,8 @@ export { default as preloadSources } from './preloadSources'
 export { default as setEditingValue } from './setEditingValue'
 export { default as setInvalidState } from './setInvalidState'
 export { default as updateSplitPosition } from './updateSplitPosition'
+export { default as pinToTop } from './pinToTop'
+export { default as pinToBottom } from './pinToBottom'
```

Comment:
Let's combine these into a single, parameterized reducer. Possible input values may be `'top'`, `'bottom'`, or `null`. `null` can be used to remove pins.